src/renderer/mod.rs

```diff
@@ -47,6 +47,7 @@ pub struct RendererSettings {
     floating_blur_amount_y: f32,
     floating_shadow: bool,
     floating_z_height: f32,
+    no_multigrid_zindex: u64,
```

Comment:
Since a user might want to have this apply to multiple zindices, how about making this a list of indices instead?

Reply:
Maybe it would make sense to say that all zindex values below the configured one will be treated as flat?

Tbh I kinda think a name like `flatten_floating_windows_below_zindex` or something more concise but in that direction might help communicate what the setting actually does. Multigrid is a feature for neovim guis that might not make sense to end users.

Reply:
> Maybe it would make sense to say that all zindex values below the configured one will be treated as flat?
> 
> Tbh I kinda think a name like `flatten_floating_windows_below_zindex` or something more concise but in that direction might help communicate what the setting actually does. Multigrid is a feature for neovim guis that might not make sense to end users.

There is a usage scenario where a floating window can be used as a preview and can do some editing. Generally, the zindex of this type of window is set very low.
For example, I set glance.nvim's zindex to 10 and treesitter-context's zindex to 20, and keep glance's preview's shadow like this:
![image](https://github.com/neovide/neovide/assets/97848247/2e9a02cd-abc7-45e4-8ce8-470104f8c54b)
That being said, a specified zindex would allow this kind of customization.